formats/json/jsTest/src/kotlinx/serialization/json/JsonDynamicImplicitNullsTest.kt

```diff
@@ -0,0 +1,14 @@
+package kotlinx.serialization.json
+
+import kotlinx.serialization.KSerializer
+
+class JsonDynamicImplicitNullsTest : AbstractJsonImplicitNullsTest() {
+    override fun <T> Json.encode(value: T, serializer: KSerializer<T>): String {
+        return JSON.stringify(encodeToDynamic(serializer, value))
```

Comment:
Hm, i see, we do not have dynamic in json test base. Maybe we'll have to unify it, but later